packages/core-ethereum/src/ethereum.ts

```diff
@@ -178,10 +176,18 @@ export async function createChainWrapper(
     nonceLock.releaseLock()
 
```

Comment:
two more lines here
```
      deferredListener.reject()
      transactions.remove(transaction.hash)
```

Reply:
why do we remove it from transactions if it's still pending? which LOC should the 2 lines go?

Reply:
Euh... referred to a wrong line. I intended to mention L.172. In the `catch (error)` block.
Listener should be resolved and remove hash from tx-manager

Reply:
thx, added